soda/bigquery/soda/data_sources/bigquery_data_source.py

```diff
@@ -17,11 +18,11 @@
 class DataSourceImpl(DataSource):
     TYPE = "bigquery"
 
-    SCHEMA_CHECK_TYPES_MAPPING: Dict = {
+    SCHEMA_CHECK_TYPES_MAPPING: dict = {
```

Comment:
yeah `dict` and `type1 | type2` are more recent. We have a task for updating the contributing.md, I added a comment about this there

Reply:
Adding `from __future__ import annotations` because I was using the new versions of annotations in some code introduced in that file, lead to one of our formatters (maybe pyupgrade) to then change them all.